### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by Angel3245**

### CODE REVIEW

File: locales/ES_es.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	"credits": {
+		"title": "Equipo",
+		"text": "Conoce al equipo tras el proyecto",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Líder del proyecto y desarrollador principal",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
+			},
```
Comment (mrjvs) [inline locales/ES_es.json:77]: jon has a new username on github, please change to this:
```suggestion
			{
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Líder del proyecto y desarrollador principal",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
			},
```

**Review by mrjvs — CHANGES_REQUESTED** [review]

**Review by mrjvs — APPROVED** [review]

**Comment by mrjvs** [comment]
great, thanks a lot

**Comment by ashmonty** [comment]
Hey @Angel3245, I'm Ash from the Pretendo Network developer team.

I'm contacting you because you have contributed to our `PretendoNetwork/website` repository in the past, and we're working on relicensing it under the AGPL3 license (https://choosealicense.com/licenses/agpl-3.0/), which protects the code from being used in closed-source projects, while still permitting modification, as long as the source code is made available. To do so, we need to ask every contributor to also relicense their contributions to AGPL3.

Do you agree to relicense your contribution(s) under AGPL3? (Yes/No)

P.S: I'm asking you here because I wasn't sure where to reach you otherwise

**Comment by Angel3245** [comment]
> Hey @Angel3245, I'm Ash from the Pretendo Network developer team.
> 
> I'm contacting you because you have contributed to our `PretendoNetwork/website` repository in the past, and we're working on relicensing it under the AGPL3 license (https://choosealicense.com/licenses/agpl-3.0/), which protects the code from being used in closed-source projects, while still permitting modification, as long as the source code is made available. To do so, we need to ask every contributor to also relicense their contributions to AGPL3.
> 
> Do you agree to relicense your contribution(s) under AGPL3? (Yes/No)
> 
> P.S: I'm asking you here because I wasn't sure where to reach you otherwise

Yes

**Pull request merged**